client/app/scripts/superdesk-archive/directives.js

```diff
@@ -323,15 +328,63 @@ define([
                                 } else {
                                     figure.html(img);
                                 }
+                                _calcRatio();
                             };
 
                             img.onerror = function() {};
                             img.src = href;
                         }
                     });
+
+                    var stopRatioWatch = scope.$watch('ratio', function(val) {
```

Comment:
Because it's not using just scope.ratio, but also image ratio to get whether its portrait or not.